pop_up.js

```diff
@@ -0,0 +1,262 @@
+    image: 'images/tonicSmall.svg',
+    image2: 'images/tonic4.svg',
+    description: `  A smart assistant to make driving more safe, efficient, and fun by
+                            unlocking your most expensive computer: your car.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['UBER', 'Lead Developer', '2018'],
+  },
+];
+
+const portfolioArticle = document.getElementById('portfolio');
+portfolioArticle.innerHTML = projectCards.map((project, index) => `
+  <article class="card ${index % 2 !== 0 ? 'card--two' : 'card-one'}">
+        <img class="card__image" src="${project.image}" alt="Tonic image" />
+        <img
+          class="card__image2 image-scale"
+          src="${project.image2}"
+          alt="Nature image"
+        />
+        <h2 class="card__title">${project.name}</h2>
+        <div class="card-items">
+          <h2 class="title__desk">${project.name2}</h2>
+          <div class="card__subtitle">
+            <h3>${project.specifics[0]}</h3>
+            <img src="images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics[1]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics[2]}</h3>
+          </div>
+          <div class="subtitle__desk">
+            <h3>${project.specifics2[0]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics2[1]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics2[2]}</h3>
+          </div>
+          <div class="card__body">
+            <p class="para-mobile">
+            ${project.description}
+            </p>
+            <p class="para-desk move-text">
+             ${project.description}
+            </p>
+            <ul class="categories">
+              <li>${project.technologies[0]}</li>
+              <li>${project.technologies[1]}</li>
+              <li>${project.technologies[2]}</li>
+            </ul>
+            <ul class="categories-desk">
+               <li>${project.technologies[0]}</li>
+              <li>${project.technologies[1]}</li>
+              <li>${project.technologies[2]}</li>
+            </ul>
+            <button data-modal="#modal" class="btn" type="button">
+              See Project
+            </button>
+          </div>
+        </div>
+ </article>
+  `).join('');
+
+/* Modal Window */
+const openModelbtn = document.querySelectorAll('[data-modal]');
+const closeModelbtn = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+const titleModal = document.querySelector('.title');
+const imageModal = document.querySelector('.modal-img');
+const descriptionModal = document.querySelector('.modal-desc p');
+const categoriesModal = document.querySelector('.modal-categories');
```

Comment:
- [ ] In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. I can see that the base projects section data is populated with the data from the `projectCards`  array a good job! 👏🏻 , but the Modal popup data is still not being created with Javascript.  Please use Javascript to dynamically render your Modals or Popup windows section as you did with the base projects section page.

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `projectCards` array. 😉 👍🏻 
 
 - (Note: the works section and the Modal popup section must be removed from HTML and generated with javascript )



https://github.com/Kel-nana/Portfolio-finish-mobile-version/blob/a3374d2a60c6fe9d4210e63cdf096fe1de00bd20/index.html#L91-L131